lib/DDG/Spice/Homebrew.pm

```diff
@@ -0,0 +1,33 @@
+package DDG::Spice::Homebrew;
+# ABSTRACT: Returns formula information from Homebrew formulas index (http://brewformulas.org).
+
+use strict;
+use DDG::Spice;
+
+primary_example_queries "brew wget";
+secondary_example_queries "homebrew wget", "wget homebrew", "wget brew";
+description "Homebrew formulas information";
+name "Homebrew";
+code_url "https://github.com/duckduckgo/zeroclickinfo-fathead/tree/master/share/fathead/homebrew";
+icon_url "/i/brewformulas.org.ico";
+topics "computing", "geek", "sysadmin";
+category "software";
+
+attribution email => ['contact@sgentile.it', 'Salvatore Gentile'],
+            github => ["https://github.com/SalGnt", "SalGnt"],
+            twitter => '_sgentile',
+            web => ['http://sgentile.it', 'Salvatore Gentile'];
+
+triggers startend => 'brew', 'homebrew';
+triggers start => 'brew install';
+
+spice to => 'http://brewformulas.org/$1.json';
```

Comment:
Is there an API limit? I think it would be best to cache the results for a month https://github.com/duckduckgo/zeroclickinfo-spice/search?utf8=%E2%9C%93&q=30d as well.